src/affiliations/models.py

```diff
@@ -2,12 +2,58 @@
 
 # Third-party dependencies:
 from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class AffiliationStatus(models.TextChoices):  # pylint: disable=too-many-ancestors
+    """Creating choices for status."""
+
+    APPLYING = "APPLYING", _("Applying")
```

Comment:
Out of curiosity, are these [tuples](https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences)? What is this `_("Some String")` syntax? I've never seen it before.

Reply:
Apparently this is an internationalization thing: https://docs.python.org/3/library/gettext.html#class-based-api.

Reply:
I followed the documentation listed here: https://docs.djangoproject.com/en/5.1/ref/models/fields/#enumeration-types Essentially: ```Django supports adding an extra string value to the end of this tuple to be used as the human-readable name, or label. The label can be a lazy translatable string. ```

There are other ways to implement this without using the tuples/enumeration types, if you would prefer. Let me know if you'd like me to refactor this.

Reply:
No need to refactor! I was just curious.